src/components/input/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ChangeEventHandler, FC, FocusEventHandler } from 'react';
+import { Input as ChakraInput } from '@chakra-ui/react';
+
+import { useDebouncedOnChange } from '../hooks';
+
+type Props = {
+  placeholder?: string;
+  isInvalid?: boolean;
+  isDisabled?: boolean;
+  size?: 'md' | 'lg';
+  onBlur?: FocusEventHandler<HTMLInputElement>;
+  onFocus?: FocusEventHandler<HTMLInputElement>;
+  onChange?: ChangeEventHandler<HTMLInputElement>;
```

Comment:
I already asked this on the checkbox PR. Shouldn't value and onChange be mandatory props since we always want a controlled input? Maybe I misunderstand something though

Reply:
it's completely fair game to have an uncontrolled input and an onChange handler (think tracking)